scrapyjs/middleware.py

```diff
@@ -109,6 +115,9 @@ def process_request(self, request, spider):
             headers=new_headers,
         )
 
+        if self.splash_auth:
+            req_rep.headers['Authorization'] = self.splash_auth
```

Comment:
Hm, and why is it better than using [HttpAuthMiddleware](http://doc.scrapy.org/en/latest/topics/downloader-middleware.html#scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware)?